src/energyplus/ForwardTranslator/ForwardTranslateAirTerminalDualDuctVAV.cpp

```diff
@@ -101,6 +101,17 @@ namespace energyplus {
       }
     }
 
+    // MinimumAirFlowTurndownScheduleName
+    boost::optional<Schedule> minAirFlowTurndownSchedule = modelObject.minimumAirFlowTurndownSchedule();
+
+    if (minAirFlowTurndownSchedule) {
+      boost::optional<IdfObject> _minAirFlowTurndownSchedule = translateAndMapModelObject(minAirFlowTurndownSchedule.get());
+
+      if (_minAirFlowTurndownSchedule && _minAirFlowTurndownSchedule->name()) {
+        idfObject.setString(AirTerminal_DualDuct_VAVFields::MinimumAirFlowTurndownScheduleName, _minAirFlowTurndownSchedule->name().get());
```

Comment:
I'd prefer doing this, but this is kind of a nitpick. if the schedule is translated, we KNOW it has a name so the check is redundant and (minimally) hurting performance

```suggestion
      if (_minAirFlowTurndownSchedule) {
        idfObject.setString(AirTerminal_DualDuct_VAVFields::MinimumAirFlowTurndownScheduleName, _minAirFlowTurndownSchedule->nameString());
```